.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: nuget restore EasyPost.sln
         # Build the test project
       - name: Build Solution
-        run: msbuild ${{ steps.test_project.outputs.test_file }}\${{ steps.test_project.outputs.test_file }}.csproj /p:platform="Any CPU" /p:configuration="Test" /p:outputPath="bin/Test" /p:target="Rebuild"
+        run: msbuild ${{ steps.test_project.outputs.test_file }}\${{ steps.test_project.outputs.test_file }}.csproj /p:platform="Any CPU" /p:configuration="Test" /p:outputPath="bin/Test" /p:target="Rebuild" -restore
```

Comment:
Was this rebased properly? If so this shouldn't be here as it was included in @jchen293's PR.

Reply:
Sorry, accidentally rebased it on the wrong branch. Fixed.